docs/mmctl_ldap_idmigrate.rst

```diff
@@ -9,7 +9,12 @@ Synopsis
 ~~~~~~~~
 
 
-Migrate LDAP IdAttribute to new value. Run this utility then change the IdAttribute to the new value.
+Migrate LDAP "IdAttribute" to a new value. Run this utility to change the value of your ID Attribute without your users losing their accounts. After running the command you can change the ID Attribute to the new value in the System Console. For example, if your current ID Attribute was "sAMAccountName" and you wanted to change it to "objectGUID", you would:
+
+1. Wait for an off-peak time when your users won’t be impacted by a server restart.
```

Comment:
Wondering if we should add a command to make server readonly for a while? It can come handy in such conditions. This comment is not so related with the PR tho 😅

Reply:
Totally agree with you. Read-only was a feature we discussed at some point in Server, but never tackled. I think it would be very useful for multiple cases, long migrations even :)